tft_bot/constants.py

```diff
@@ -14,6 +14,10 @@
             "game": r"\Game\League of Legends.exe",
         },
     },
+    "window_titles": {
+        "game": "League of Legends (TM) Client",
+        "client": "League of Legends",
+    },
```

Comment:
Heh, I had been lazy by not putting those into the constants file, so this is appreciated cleanup 🧹